build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -175,6 +188,16 @@ class SeleniumWebDriverController {
         this.getWaitFn_(() => webElement.getText()));
   }
 
+  /**
+   * @param {!ElementHandle<!WebElement>} handle
+   * @return {!Promise<string>}
+   * @override
+   */
+  getElementTagName(handle) {
+    const webElement = handle.getElement();
+    return webElement.getTagName();
```

Comment:
Should this return a `ControllerPromise` as defined in the interface?

Reply:
Ah my bad, the interface should not be a `ControllerPromise` since an element's `tagName` cannot change.